iceoryx_posh/roudi_env/include/iceoryx_posh/roudi_env/minimal_roudi_config.hpp

```diff
@@ -14,15 +14,15 @@
 //
 // SPDX-License-Identifier: Apache-2.0
 
-#ifndef IOX_POSH_ROUDI_ENVIRONMENT_MINIMAL_ROUDI_CONFIG_HPP
-#define IOX_POSH_ROUDI_ENVIRONMENT_MINIMAL_ROUDI_CONFIG_HPP
+#ifndef IOX_POSH_ROUDI_ENV_MINIMAL_ROUDI_CONFIG_HPP
+#define IOX_POSH_ROUDI_ENV_MINIMAL_ROUDI_CONFIG_HPP
 
 #include "iceoryx_posh/iceoryx_posh_config.hpp"
 #include "iox/builder.hpp"
 
 namespace iox
 {
-namespace testing
+namespace roudi_env
```

Comment:
I would also be fine to keep the `testing` namespace but it felt somehow better to move the `RouDiEnvironment` to its own namespace